server/statistics/hot_peer.go

```diff
@@ -26,30 +27,70 @@ const (
 	dimLen
 )
 
+type dimStat struct {
+	typ         int
+	Rolling     *movingaverage.TimeMedian
+	LastAverage *movingaverage.AvgOverTime
```

Comment:
what's the difference between the two?

Reply:
`LastAverage ` is used to control `hot-degree` and `anti-count`, `Rolling` is used to statistic flow.